src/NuGet/BuildNuGets.csx

```diff
@@ -122,6 +124,8 @@ string[] TestPackageNames = {
 var PreReleaseOnlyPackages = new HashSet<string>
 {
     "Microsoft.CodeAnalysis.Build.Tasks",
+    "Microsoft.CodeAnalysis.CSharp.CodeStyle",
+    "Microsoft.CodeAnalysis.VisualBasic.CodeStyle",
```

Comment:
We are planning to publish these on NuGet. Does that mean I should remove them from `PreReleaseOnlyPackages` from the start (i.e. before they would actually be considered "ready for public release")?